test/linter.js

```diff
@@ -80,7 +80,7 @@ async function getTagsFromConfig(config) {
 }
 
 function execLinterCommand(args) {
-  var cmd = `npx autorest@2.0.4243 --validation --azure-validator --message-format=json ${args}`.trim();
```

Comment:
Yes, this is ok; the `--version` sets the autorest-core version (where as `npx autorest@2.0.4243` chooses the autorest bootstrapper version.)